### PR TITLE
Get miner/sharder stats API tests

### PR DESCRIPTION
**Pull request by stewartie4**
Add simple API tests for the /v1/miner/stats/get and /v1/sharder/stats/get endpoints 
docs also updated:
image

### CODE REVIEW

File: tests/api_tests/get_stats_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		require.Greater(t, stats.AverageBlockSize, 0, httpResponse)
+		require.NotNil(t, stats.NetworkTime, httpResponse)
+	})
+
+	t.Run("Get sharder stats call should return successfully", func(t *testing.T) {
+		t.Parallel()
+		stats, httpResponse, err := v1SharderGetStats(t, util.ConsensusByHttpStatus("200 OK"))
+
+		require.Nil(t, err)
+		require.Equal(t, "200 OK", httpResponse.Status(), httpResponse)
```
Comment (YarikRevich) [inline tests/api_tests/get_stats_test.go:35]: It's better to replace manually written "200 OK" to some constant like "STATUS_200"

File: tests/api_tests/get_stats_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+func TestGetStats(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Get miner stats call should return successfully", func(t *testing.T) {
+		t.Parallel()
+		stats, httpResponse, err := v1MinerGetStats(t, util.ConsensusByHttpStatus("200 OK"))
+
+		require.Nil(t, err)
+		require.Equal(t, "200 OK", httpResponse.Status(), httpResponse)
```
Comment (YarikRevich) [inline tests/api_tests/get_stats_test.go:17]: It's better to replace manually written "200 OK" to some constant like "STATUS_200"

**Review by YarikRevich — CHANGES_REQUESTED** [review]

**Review by Kishan-Dhakan — APPROVED** [review]

File: internal/api/util/zerochain.go [hunk 3]
```diff
@@ -39,18 +39,31 @@ func (z *Zerochain) Init(config Config) {
 	z.Sharders = healthySharders
 }
 
-func (z *Zerochain) GetFromMiners(t *testing.T, endpoint string, consensusMet ConsensusMetFunction) (*resty.Response, error) { //nolint
+func (z *Zerochain) GetFromMiners(t *testing.T, endpoint string, consensusMet ConsensusMetFunction, targetObject interface{}) (*resty.Response, error) { //nolint
```
[the remark this answers is not in the document]
Comment (stewartie4) [reply internal/api/util/zerochain.go:42]: "dst" seems distinctly unreadable to me - what does "dst" stand for? I think targetObject is clear in that it tells you what this param is and it's purpose

File: tests/api_tests/execute_faucet_test.go [hunk 4]
```diff
@@ -95,7 +94,7 @@ func getBalanceWithoutAssertion(t *testing.T, clientId string) (*model.Balance,
 
 func executeFaucet(t *testing.T, wallet *model.Wallet, keyPair model.KeyPair) *model.TransactionResponse {
 	t.Logf("Executing faucet...")
-	txnDataString, err := json.Marshal(model.SmartContractTxnData{Name: "pour"})
+	_, err := json.Marshal(model.SmartContractTxnData{Name: "pour"})
```
[the remark this answers is not in the document]
Comment (stewartie4) [reply tests/api_tests/execute_faucet_test.go:97]: you're right, I found the same but forgot to remove this

File: tests/api_tests/get_stats_test.go (hunk 2 above)
Comment (stewartie4) [reply tests/api_tests/get_stats_test.go:17]: Agreed

**Pull request merged**